src/main/java/controller/CarRaceController.java

```diff
@@ -0,0 +1,35 @@
+package controller;
+
+import model.Car;
+import model.CarRaceGame;
+import model.CarRandomNumberGenerator;
+import view.InputView;
+import view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarRaceController {
+    private final CarRandomNumberGenerator carRandomNumberGenerator = new CarRandomNumberGenerator();
+
+    public void start() {
+        List<String> carNames = InputView.inputCarNames();
+        int roundCount = InputView.inputRoundCount();
+        List<Car> cars = new ArrayList<>();
+
+        for (String carName : carNames) {
+            cars.add(new Car(carName, carRandomNumberGenerator));
+        }
+
+        CarRaceGame carRaceGame = new CarRaceGame(cars);
+
+        OutputView.printResultTitle();
+
+        for (int i = 1; i <= roundCount; i++) {
+            List<Car> currentCars = carRaceGame.playSingleRoundAndGetCars();
+            OutputView.printRaceStatus(currentCars);
+        }
+
+        OutputView.printWinners(carRaceGame.getWinners());
+    }

```

Comment:
자동차 객체생성에서 유효성 검사를 하고 있으니까 이쪽에 try-catch 넣으면 될듯? 

```suggestion
       try {
            List<String> carNames = InputView.inputCarNames();
            int roundCount = InputView.inputRoundCount();
            
            List<Car> cars = carNames.stream()
                    .map(name -> new Car(name, carRandomNumberGenerator))
                    .collect(Collectors.toList());
                    
            // ... 게임 진행 로직
            
        } catch (IllegalArgumentException e) {
            System.out.println(e.getMessage());
            // 재시도 로직 추가 (do-while 사용하는것밖에 생각안나넹;;ㅎ
        }
```

Reply:
로직을 전체적으로 바꿔서
- 유효성 검사 및 예외 throw는 `view`에서 validator 클래스에 의해 실행
- 예외를 catch하는 작업은 `controller`

에서 실행되도록 바꿔보았습니다